cmd/clusterctl/client/client_test.go

```diff
@@ -300,7 +301,7 @@ func (f fakeClusterClient) ProviderInstaller() cluster.ProviderInstaller {
 }
 
 func (f *fakeClusterClient) ObjectMover() cluster.ObjectMover {
-	if f.fakeObjectMover == nil {
+	if util.IsNil(f.fakeObjectMover) {
```

Comment:
Not sure why, but I started getting hit with errors about how this comparison was not possible :thinking: